src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTab.java

```diff
@@ -53,9 +58,23 @@ public JournalAbbreviationsTab() {
     private void initialize() {
         viewModel = new JournalAbbreviationsTabViewModel(preferencesService, dialogService, taskExecutor, abbreviationRepository);
 
+        filteredAbbreviations = new FilteredList<>(viewModel.abbreviationsProperty(), s -> true);
+
         setButtonStyles();
         setUpTable();
         setBindings();
+
+        searchBox.textProperty().addListener((observable, previousText, newText) -> {
+            if (newText.isEmpty()) {
+                filteredAbbreviations.setPredicate(s -> true);
```

Comment:
```suggestion
                filteredAbbreviations.setPredicate(Boolean::True);
```

Reply:
Hm, Boolean::True does not seem to exist. The closest thing I found that you might have meant is BooleanSupplier::TRUE_SUPPLIER, but that is a supplier, not a predicate. What did you mean?

Reply:
I was just thinking of the Method reference, but didn't know if it existed for booleans

Reply:
Isn't skipping adding the predicate for the constructor or setting it to null correct for this use case?

Reply:
Hm. I got an e-mail that said @k3KAW8Pnf7mkmdSMPHz27 commented on this, but can't find it anywhere here on github. Basically he proposed to just not define the predicate in the constructor / setting the predicate to null.
I think keeping it as 's -> true' is easier to read and immediately clear, don't you agree?

Reply:
Yup. Sorry about that, I was using the aggravating iOS GitHub again, I really should stop doing that X).
I thought that GitHub preserved deleted comments for 24 hours before permanently removing them but I was incorrect.